projects/plugins/jetpack/extensions/blocks/premium-content/logged-out-view/logged-out-view.php

```diff
@@ -62,7 +62,7 @@ function render_loggedout_view_block( $attributes, $content, $block = null ) {
 
 	// Old versions of the block were rendering the subscribe/login button server-side, so we need to still support them.
 	if ( ! empty( $attributes['buttonClasses'] ) ) {
-		require_once '../_inc/legacy-buttons.php';
+		require_once __DIR__ . '/_inc/legacy-buttons.php';
```

Comment:
Err, shouldn't that be
```suggestion
		require_once __DIR__ . '/../_inc/legacy-buttons.php';
```

Reply:
Oops yeah, copy / paste issue, thank you!